content/rs/concepts/terminology.md

```diff
@@ -33,7 +33,7 @@ database instances, referred to as "shards".
 RS supports various database configurations:
 
 - **Standard Redis database** - A single Redis shard with no
-    replication or data sharding.
+    replication or clustering.
```

Comment:
I think the right term here is data sharding (the operation of splitting data into shards is sharding).
@AlonMagrafta ?

Reply:
same as above

Reply:
To add fuel to the fire - I think the right term is data partitioning... https://redis.io/topics/partitioning

Reply:
I was afraid to light this match ;)

Reply:
Whether the correct term is clustering (as used in the database configuration page), sharding (as used informally and in the sharding policy), or partitioning (as in redis.io) we need to decide on the correct terms, define them, and use them correctly across the product.
I'll add this to the terminology doc.
cc: @banker